README.rst

```diff
@@ -178,4 +178,5 @@ Common description. For a detailed description, use ``tt help command`` .
 * ``cat`` - print into stdout the contents of .snap/.xlog files.
 * ``play`` - play the contents of .snap/.xlog files to another Tarantool instance.
 * ``coredump`` - pack/unpack/inspect tarantool coredump.
-* ``run`` - start a tarantool instance. 
+* ``run`` - start a tarantool instance.
```

Comment:
AFAIS, the run module has nothing to do with this PR, please move this change to a separate commit for having clear history. [2]